src/blkio.rs

```diff
@@ -432,6 +436,16 @@ impl BlkIoController {
     /// Constructs a new `BlkIoController` with `root` serving as the root of the control group.
     pub fn new(root: PathBuf, v2: bool) -> Self {
         Self {
+            mount_root: Default::default(),
```

Comment:
All the controllers have the exact same code pattern. Shouldn't we try to factor out that code a little more?